js/coinflex.js

```diff
@@ -0,0 +1,2384 @@
+        //                 "toAccount": "38774",
+        //                 "id": "758113170128764931",
+        //                 "status": "COMPLETED",
+        //                 "transferredAt": "1651428178967"
+        //              }
+        //          ]
+        //     }
+        //
+        const transfers = this.safeValue (response, 'data', []);
+        return this.parseTransfers (transfers, currency, since, limit);
+    }
+
+    parseTransfer (transfer, currency = undefined) {
+        //
+        // fetchTransfers
+        //
+        //     {
+        //         "asset": "USDT",
+        //         "quantity": "5",
+        //         "fromAccount": "38432",
+        //         "toAccount": "38774",
+        //         "id": "758113170128764931",
+        //         "status": "COMPLETED",
+        //         "transferredAt": "1651428178967"
+        //      }
+        //
+        // transfer
+        //
+        //     {
+        //         "asset": "USDT",
+        //         "quantity": "5",
+        //         "fromAccount": "38422",
+        //         "toAccount": "38776",
+        //         "transferredAt": "1651704762775"
+        //     }
+        //
+        const currencyId = this.safeString (transfer, 'asset');
+        const timestamp = this.safeString (transfer, 'transferredAt');
+        const fromAccount = this.safeString (transfer, 'fromAccount');
+        const toAccount = this.safeString (transfer, 'toAccount');
+        let status = this.parseTransactionStatus (this.safeString (transfer, 'status'));
+        // if case of 'transfer', as status field is not available, 'transferredAt' prop should mean that transfer was done
+        if (status === undefined && timestamp !== undefined) {
+            status = 'ok';
+        }
+        return {
+            'id': this.safeString (transfer, 'id'),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'currency': this.safeCurrencyCode (currencyId, currency),
+            'amount': this.safeNumber (transfer, 'quantity'),
+            'fromAccount': fromAccount,
+            'toAccount': toAccount,
+            'status': status,
+            'info': transfer,
+        };
+    }
+
+    async buildOrderRequest (market, type, side, amount, price = undefined, params = {}) {
+        const clientOrderId = this.safeString (params, 'clientOrderId');
+        const maxCOI = '9223372036854775807';
+        if ((clientOrderId !== undefined) && Precise.stringGt (clientOrderId, maxCOI)) {
+            throw new InvalidOrder (this.id + ' createOrder() param clientOrderId should not exceed ' + maxCOI);
+        }
+        const orderType = this.convertOrderType (type);
+        // creating stop orders using type argument will mess up the unification logic (beacuse of missing market/limit). So, we have to use unified approach for sending stop orders
+        if (orderType === 'STOP') {
```

Comment:
I'm a bit confused here. You're throwing the exception but you did not check parameters for the required "stopPrice" so will always throw. Am I missing something? 😅

Reply:
I'll get into that tomorrow. this problem might related to the stop-orders probably 👍 i've not resolved that another review too  yet, will ping you tomorrow morning

Reply:
About this - well, my implementation might not be ideal, but i thought here we could force users not to use `STOP` as order type (because it causes logic mess-up between market/limit orders), so if users still explicitly want to use exchange specific params, then they can user `params['orderType'] = 'STOP'` , however, in our unified arguments (and especially in this case) I think it's better that user used our unified approach. 
@kroitor let me know if this is wrong and we shouldnt do it.

Reply:
yes of course, but `stopPrice` is exchange specific param too with this exchange :) so, there was nothing I could mention more there.

Reply:
> I was making an analogy between the unified params <-> exchange specific params, I was not talking about stopPrice per se that was just an example. I meant, like we support both terminologies for stopPrice,clientOrderId, etc... It is probably not wrong to do the same with the order Type

@carlos  please make commit, I will see which code/part you mean (as I doubt I misunderstood the above comments), please feel free to do

Reply:
@ttodua You just did it, 🙂  I was just suggesting allowing the users to use the "STOP" type. Going to test it now.